docs/resources/app.md

```diff
@@ -0,0 +1,48 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "lumos_app Resource - terraform-provider-lumos"
+subcategory: ""
+description: |-
+  App Resource
+---
+
+# lumos_app (Resource)
+
+App Resource
+
+## Example Usage
+
+```terraform
+resource "lumos_app" "my_app" {
+  category    = "...my_category..."
+  description = "...my_description..."
+  name        = "Frances Collier"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Optional
```

Comment:
why does it mark these as optional?

Reply:
Should all of these attributes be `required`? If so we should mark that in `apiserver`, because the [API docs](https://developers.lumos.com/reference/createapp) also have these as optional